corehq/apps/reports/templates/reports/partials/download_export.html

```diff
@@ -0,0 +1,48 @@
+{% load i18n %}
+{% load hq_shared_tags %}
```

Comment:
it was easier to pull out the code I needed, and when this goes live I'll delete the other code